static/js/browser2/statvar_node.tsx

```diff
@@ -71,4 +93,14 @@ export class StatVarNode extends React.Component<
       </Collapsible>
     );
   }
+
+  private onClickStatVarLink = () => {
+    const uri = URI_PREFIX + this.props.statVar.id;
+    window.open(uri);
```

Comment:
this triggers a pop-up warning because it's no longer the result of a user action. could you add a todo to improve this? (on suggestion is to save the context of the window, and update the URL when the result comes back).

Reply:
I think the pop-up warning was in the redirectToObsPage in observationChart component, so I added a todo there